components/Pagination.tsx

```diff
@@ -69,6 +101,19 @@ export default function Pagination(props: Props) {
           disabled={isFirstPage}
           onClick={goToPreviousPage}
         />
+        <Input
```

Comment:
With the introduction of page number input do you think we should also display the total page numbers? Currently the display to the left of the arrow key shows row numbers (e.g. 9-10 of 23), but I think it would be intuitive to also show something like page 5 of 5.

<img width="871" alt="image" src="https://github.com/user-attachments/assets/602598e0-83af-4ec1-96f1-7b604b1566bb" />

Reply:
<img width="781" alt="image" src="https://github.com/user-attachments/assets/1db31e7b-a6e3-42c2-b634-c6daaf9ba7a0" />

Reply:
<img width="758" alt="image" src="https://github.com/user-attachments/assets/27ddfa69-053a-4cb4-ae29-37e918772862" />

Reply:
Added some spacing here:
![Uploading image.png…]()